thirtyinch/src/main/java/net/grandcentrix/thirtyinch/internal/ActivityScopedPresenters.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2017 grandcentrix GmbH
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.grandcentrix.thirtyinch.internal;
+
+
+import net.grandcentrix.thirtyinch.TiLog;
+import net.grandcentrix.thirtyinch.TiPresenter;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Simple wrapper around a {@link HashMap} to save {@link TiPresenter} by id. For every {@link
+ * android.app.Activity} containing a {@link TiPresenter} a corresponding {@link
+ * ActivityScopedPresenters} will be created
+ */
+public class ActivityScopedPresenters {
```

Comment:
I think about implementing `Map` directly or extending from `HashMap`. But not needed now. So keep it like it is ;)